### ISSUE TITLE
Crash at start when werkzeug > 0.13 

### ISSUE THREAD

**Issue by Roshless** [ptpb/pb#208]
Arch Linux has 0.14.1 in repo, I don't know if it also affects 0.14

```
Feb 11 13:31:05 destruction uwsgi[27759]: Traceback (most recent call last):
Feb 11 13:31:05 destruction uwsgi[27759]:   File "/usr/lib/python3.6/site-packages/pb/__main__.py", line 35, in <module>
Feb 11 13:31:05 destruction uwsgi[27759]:     app = create_app()
Feb 11 13:31:05 destruction uwsgi[27759]:   File "/usr/lib/python3.6/site-packages/pb/pb.py", line 75, in create_app
Feb 11 13:31:05 destruction uwsgi[27759]:     app.url_map.update()
Feb 11 13:31:05 destruction uwsgi[27759]:   File "/usr/lib/python3.6/site-packages/werkzeug/routing.py", line 1358, in update
Feb 11 13:31:05 destruction uwsgi[27759]:     self._rules.sort(key=lambda x: x.match_compare_key())
Feb 11 13:31:05 destruction uwsgi[27759]:   File "/usr/lib/python3.6/site-packages/werkzeug/routing.py", line 1358, in <lambda>
Feb 11 13:31:05 destruction uwsgi[27759]:     self._rules.sort(key=lambda x: x.match_compare_key())
Feb 11 13:31:05 destruction uwsgi[27759]:   File "/usr/lib/python3.6/site-packages/pb/routing.py", line 32, in match_compare_key
Feb 11 13:31:05 destruction uwsgi[27759]:     return not bool(self.namespace_only), bool(self.arguments), -len(self._weights), self._weights
Feb 11 13:31:05 destruction uwsgi[27759]: AttributeError: 'Rule' object has no attribute '_weights'
Feb 11 13:31:05 destruction uwsgi[27759]: unable to load app 0 (mountpoint='') (callable not found or import error)
```

**Comment by buhman** [comment]
Thanks, secretly every single werkzeug release breaks pb, so I'm beyond unsurprised.

**Comment by buhman** [comment]
Fixed in 2528e9a
